DXRFixes/DeusEx/Classes/ElevatorMover.uc

```diff
@@ -17,6 +18,25 @@ function bool EncroachingOn( actor Other )
     return false;
 }
 
+function float GetMoveSpeed()
+{
+    // assumes no valid KeyPos elements after the first are (0, 0, 0)
+    local float maxDist;
+    local int i, j;
+
+    if (moveSpeed == 0.0) {
+        for (i = 0; i < ArrayCount(KeyPos); i++) {
+            for (j = i + 1; j < ArrayCount(KeyPos); j++) {
+                if (KeyPos[j] == vect(0.0, 0.0, 0.0)) continue;
+                maxDist = Max(maxDist, VSize(KeyPos[i] - KeyPos[j]));
+            }
+        }
+        moveSpeed = maxDist / MoveTime;
```

Comment:
This definitely works (I gave it a quick spin on Silo), but I think using the maximum distance ends up being much too fast in that case.  I almost wonder if it the assumption should be that the original MoveTime is the assumption for two nearby stops?